vector/v.db.select/main.c

```diff
@@ -67,12 +88,15 @@ int main(int argc, char **argv)
     char *fsep, *vsep;
     struct bound_box *min_box, *line_box;
     int i, line, area, init_box, cat, field_number;
+    int plain, json, csv, vertical;
```

Comment:
Just like the consolidated `format=` option, I would use one variable for different formats because they are all exclusive.
```c
#define PLAIN 0
#define CSV 1
#define JSON 2
#define VERTICAL 3
...

int format;
```

Reply:
Done in d64c9df56 as an enum.